test/k8sT/hubble.go

```diff
@@ -198,6 +198,20 @@ var _ = Describe("K8sHubbleTest", func() {
 			res.ExpectContains(`hubble_flows_processed_total{protocol="TCP",subtype="to-endpoint",type="Trace",verdict="FORWARDED"}`)
 		})
 
+		It("Test TLS certificate", func() {
+			cmd := "cat /var/lib/cilium/tls/hubble/server.crt"
+			res := kubectl.ExecPodCmd(helpers.CiliumNamespace, ciliumPodK8s1, cmd)
+			res.ExpectSuccess("%q cannot read the hubble server TLS certificate", ciliumPodK8s1)
```

Comment:
Nit: These msgs are usually used to grep to the test code, so having a format specifier will throw it off. Can we generalize the msg to say "Cilium pod cannot read ..."?

Reply:
Make sense, thanks!